static/sass/components/_charts.scss

```diff
@@ -0,0 +1,33 @@
+// SVG Charts
```

Comment:
well, I moved it from icons which was really wrong, into a slightly less wrong place. It is specific to the grids in that post, but I am thinking it could maybe be reused if we are using a line-based grid SVG somewhere else? I think I will be writing more CSS Grid posts so maybe it can stay put for now?